draft-ietf-quic-transport.md

```diff
@@ -656,13 +655,15 @@ version. The resent packets MUST use new packet numbers.  These packets MUST
 continue to have the VERSION flag set and MUST include the new negotiated
 protocol version.
 
-The client MUST include its selected version on all packets until it starts
-protecting packets with 1-RTT keys.  Only unprotected packets and 0-RTT
-protected packets can include a version.  A client MUST NOT change the version
-it uses unless it is in response to a version negotiation packet from the
-server.
+The client MUST set the VERSION flag on all packets until version negotiation 
+concludes. Version negotiation successfully concludes when the client receives a
+packet from the server with the VERSION flag unset. All subsequent packets sent
+by the client MUST have the VERSION flag unset.
```

Comment:
It is absolutely enforceable.  Unless your point is that the presence of absence of the version field isn't authenticated, in which case I guess I'd have to concede that you don't want an attacker ADDING a version to your packet so that you would kill the connection.

If we are going with a SHOULD here, we need to explain why (and the consequences if the SHOULD is not respected, I'll open a small PR).